client/tests.json

```diff
@@ -33,6 +33,30 @@
 			},
 			"test": [ "assembler", "index", "store" ]
 		},
+		"reader-comment-email-subscriptions": {
```

Comment:
Ah, that makes sense. I was also thinking about making all of these `index` but ... :chicken: